lib/db/mysql/patches/patch-021-022.sql

```diff
@@ -0,0 +1,8 @@
+-- Add column to stash the `profileChangedAt` value
+ALTER TABLE codes ADD COLUMN profileChangedAt BIGINT DEFAULT 0,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+ALTER TABLE tokens ADD COLUMN profileChangedAt BIGINT DEFAULT 0,
+ALGORITHM = INPLACE, LOCK = NONE;
```

Comment:
We may also want to add it to the `refreshTokens` table, because a refreshToken can be used to create a new accessToken, and we'll want to new accessToken to have a sensible value for `profileChangedAt`.